Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      isArray: false
+      name: disabled
+      required: false
+    - default: false
+      description: Negate source
+      isArray: false
+      name: negate_source
+      required: false
+    - default: false
+      description: Negate the destination.
+      isArray: false
+      name: negate_destination
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
+      description: This argument specifies rather to append or overwrite the....
+      isArray: false
+      name: overwrite
+      predefined:
+        - 'true'
+        - 'false'
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Update an existing security rule.
+    execution: false
+    name: prisma-sase-security-rule-update
+    outputs:
+    - contextPath: PrismaSase.SecurityRule
+      description: Edited security rule.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.action
+      description: Security rule action.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.application
+      description: Security rule application.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.category
+      description: Security rule category.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.description
+      description: Security rule description.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.destination
+      description: Security rule destination.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.folder
+      description: Security rule folder.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.from
+      description: Security rule from field (source zone(s)).
+      type: String
+    - contextPath: PrismaSase.SecurityRule.id
+      description: Security rule ID.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.name
+      description: Security rule name.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.profile_setting.group
+      description: Security rule profile setting.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.service
+      description: Security rule service.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.source
+      description: Security rule source.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.source_user
+      description: Security rule source user.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.to
+      description: Security rule to field (destination zone(s)).
+      type: String
+  - arguments:
+    - default: false
+      description: ID of the address object to edit.
+      isArray: false
+      name: object_id
+      required: true
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is it the folder location or the type of folder?